test/utils/token_set_utils_test.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:design_tokens_builder/utils/token_set_utils.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('Get token sets', () {
+    test('succeeds', () {
+      final result = getTokenSets({
+        '\$metadata': {
+          'tokenSetOrder': ['global', 'light', 'dark'],
```

Comment:
So global is always a default token set? Is that something normal in the design token generator?